.changeset/two-carrots-poke.md

```diff
@@ -0,0 +1,7 @@
+---
+'@compiled/css': minor
+---
+
+[BREAKING] Add a deterministic sorting to media queries and other at-rules in Compiled. We use a simplified version of what the [`sort-css-media-queries`](https://github.com/OlehDutchenko/sort-css-media-queries?tab=readme-ov-file#mobile-first) package does - sorting `min-width` and `min-height` from smallest to largest, then `max-width` and `max-height` from largest to smallest.
```

Comment:
Do we need to release a breaking version of the bundler / babel plugins as well?

Reply:
hmm that's a good point 🤔 yeah i'll add the bundler plugins and babel plugin to this

Reply:
Does `[BREAKING]` do anything special?  To me `breaking === major`, but I won't nitpick anything here, some of these might even be `0.x` where `minor === major` anyways…just thinking aloud.

Reply:
not really, it's just flagging that something _really_ can break